src/commands.rs

```diff
@@ -4925,7 +4925,11 @@ fn cmd_git_clone(
 
     let (workspace, repo) = Workspace::init_internal_git(ui.settings(), wc_path.clone())?;
     let git_repo = get_git_repo(repo.store())?;
-    writeln!(ui, "Fetching into new repo in {:?}", wc_path)?;
+    writeln!(
+        ui,
+        r#"Fetching into new repo in "{}""#,
+        wc_path.to_string_lossy()
```

Comment:
probably better to use `wc_path.display()`.

(or maybe we can switch to camino::Utf8Path unless we do need to support weird file name encoding.)